docs/tutorials/mobile/index.md

```diff
@@ -9,73 +9,103 @@ redirect_from: /get-started/with-mobile
 
-   If you are starting from scratch, bootstrap your mobile application according in your mobile framework XCode or Android Development Kit. TODO check this.
+### Measure the application's performance
 
-   a. Add the ONNX Runtime dependency
-  
-   b. Consume the onnxruntime API in your application
-   
-   c. Add pre and post processing appropriate to your application and model
+Measure the application's performance against the requirements of your target platform. This includes:
 
-4. How do I optimize my application?
+* application binary size
+* model size
+* application latency
+* power consumption
 
-   **To reduce binary size:**  Use the ONNX Runtime mobile package or a custom build to reduce the binary size. The mobile package requires use of an ORT format model.
+If the application does not meet its requirements, there are optimizations that can be applied.
 
-   **To reduce memory usage:** Use an ORT format model as that uses less memory.
+### Optimize your application
+
+#### Reduce model size
+
+One method of reducing model size is to quantize the model. This reduces an original model with 32-bit weights by approximately a factor of 4, as the weights are reduced to 8-bit.
+
+Another way of reducing the model size is to find a new model with the same inputs, outputs and architecture that has already been optimized for mobile. For example: MobileNet and MobileBert.
+
+#### Reduce application binary size
+
+There are two options for reducing the ONNX Runtime binary size.
+
+1. Use the published packages that are optimized for mobile
+
+   * Android Java/C/C++: onnxruntime-mobile
+   * iOS C/C++: onnxruntime-mobile-c
+   * iOS Objective-C: onnxruntime-mobile-objc
+
+   These mobile packages have a smaller binary size but limited feature support, like a reduced set of operator implementations and the model must be converted to [ORT format](../../reference/ort-format-models.md#convert-onnx-models-to-ort-format).
+
+   See the [install guide](../../install/#install-on-web-and-mobile) for package specific instructions.
+
+   If the mobile package does not have coverage for all of the operators in your model, then you can build a custom runtime binary based your specific model.
+
+2. Build a custom runtime based on your model(s)
+
+   One of the outputs of the ORT format conversion is a build configuration file, containing a list of operators from your model(s) and their types. You can use this configuration as input to the custom runtime binary build script.
+
+   The process to build a [custom runtime](../../build/custom.md) uses the same build scripts as standard ONNX Runtime, with some extra parameters.
+
+To give an idea of the binary size difference between mobile and full packages:
+
+ONNX Runtime 1.13.1 Android package library file size:
+
+|Package|Size|
+|-|-|
+|onnxruntime-mobile|18.8 MB|
+|onnxruntime-android|5.6 MB|
```

Comment:
I hope the mobile build isn't 3x the full one :P

Reply:
Also those sizes are the full AAR with 4 plaforms compressed. Would be better to split out maybe arm32 (will be the smallest) and arm64 separately. e.g. the arm64 mobile build is 3.2MB uncompressed and 1.2MB compressed. 

<img width="341" alt="image" src="https://user-images.githubusercontent.com/979079/215658041-8a1dd58b-379b-465e-9f80-f0e49a9a94f8.png">